src/plugins/summaryWidget/src/SummaryWidget.js

```diff
@@ -54,20 +54,25 @@ define([
 
         this.activeId = 'default';
         this.rulesById = {};
-        this.domElement = $(widgetTemplate);
-        this.toggleRulesControl = $('.t-view-control-rules', this.domElement);
-        this.toggleTestDataControl = $('.t-view-control-test-data', this.domElement);
-        this.widgetButton = this.domElement.children('#widget');
+        this.domElement = templateHelpers.convertTemplateToHTML(widgetTemplate).firstChild;
+        this.toggleRulesControl = this.domElement.querySelector('.t-view-control-rules');
+        this.toggleTestDataControl = this.domElement.querySelector('.t-view-control-test-data');
+
+        const domChildren = this.domElement.children;

```

Comment:
Theoretically there should never be more than one element in a dom with the same ID, but that's in theory :) In any case, I think we can restore the exact behavior of zepto here using a single line of code:
``` JavaScript
this.widgetButton = this.domElement.querySelector(':scope > #widget')
```